src/sbmlutils/report/sbmlreport.py

```diff
@@ -225,18 +229,18 @@ def _create_html(
             "model": model_dict(model),
             "submodels": listOfSubmodels_dict(model),
             "ports": listOfPorts_dict(model),
-            "functions": listOfFunctions_dict(model),
+            "functions": listOfFunctions_dict(model, math_type),
```

Comment:
This is okay for now, but we have to refactor the report module into a class. We have to start writing the proposal and make the design of the new SBML report. I.e., there has to be a major refactoring in new components which have to work together.